CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [1.8.0] - Unreleased
 
+- Added: [#XXXX](https://github.com/ethereum/aleth/pull/XXXX) EIP XXXX: EXTSLOAD opcode.
```

Comment:
```suggestion
- Added: [#5805](https://github.com/ethereum/aleth/pull/5805) EIP 2330: EXTSLOAD opcode.
```

Reply:
Thanks, this is great feedback, I'll make those changes.

Regarding LegacyVM.cpp vs interpreter VM.cpp I would like to understand more about their differences and future. Is there an origin story about them I can read up on?

Reply:
Not much of a story - aleth-interpreter is the same code adapted to conform to EVMC API. We keep both for now, because EVMC does not yet support all of the features we need in EVM (like tracing and individual EIP activation)